docs/clients/brushless_drive.rst

```diff
@@ -105,72 +105,116 @@ Message Table
+| 40     | motor_l                       | get, set, save | float     | H        | Cross inductance                                                                                                                     |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 41     | derate                        | get            | int32     | PU fix16 | Amount of derating. No derate = 65536, full derate = 0                                                                               |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 44     | emf                           | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 45     | volts_at_max_amps             | get            | float     |          |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 46     | slew_volts_per_second         | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 47     | slew_enable                   | get, set, save | float     | V        |                                                                                                                                      |      |
++--------+-------------------------------+----------------+-----------+----------+--------------------------------------------------------------------------------------------------------------------------------------+------+
+| 48     | motoring_supply_current_limit | get, set, save | float     | fix16_t  |                                                                                                                                      |      |
```

Comment:
These regen and motoring limiting things aren't on our stock modules currently, maybe they will be someday but we haven't really decided on that. Not sure what our policy is on that. Are we listing stock module endpoints only on the RTD since that's all about stock stuff, or are we listing the entirety of all the entries, even if they aren't on the stock. @iq-matt?

Reply:
Tough call really.  It means we don't have any documentation for the customers that do have these extra features if we leave it to stock.  The other issue is the same table is used for all motors with those clients, like 2306 and 8108 use the same table, but included features change even between those.  So either we include the minimum feature set or the maximum, but I don't think there's such a thing as a single "stock" list anymore.  

I think having the full list is the best of the bad options.  We can put an asterisk that says not all firmwares have all endpoints.

Reply:
The only other solution is to have copies of all of the tables for each module type.  This sounds like a maintenance nightmare, but might be the only way for each motor to have listed only the supported features.